ssh/validate.py

```diff
@@ -21,37 +27,31 @@ def validate_ssh_key_path(ssh_key_path):
                                               'are not allowed. Use a key without a passphrase.')
 
 
-def validate_ssh_port(ssh_port):
-    # Validate ssh port between 1 - 32000
-    assert isinstance(ssh_port, int), 'ssh port should be integer'
-    assert 1 <= ssh_port <= 32000, 'ssh port should be int between 1 - 32000'
+def validate_int_in_range(name, value, low, high):
```

Comment:
https://github.com/dcos/dcos/blob/master/gen/calc.py#L131 and this should share code